doc/developer/path-internals-pcep.rst

```diff
@@ -0,0 +1,167 @@
+PCEP Module Internals
+=====================
+
+Introduction
+------------
+
+The PCEP module for the pathd daemon is using the PCEP protocol described in
+:rfc:`5440` to update the policies and candidate paths.
+
+The protocol encoding/decoding and the basic session management is handled by
+the pceplib external library available at **TODO: TO BE DEFINED**.
+
+Together with pceplib, this module supports at least partially:
+
+  - :rfc:`5440`
+  - :rfc:`8231`
+  - :rfc:`8408`
+
```

Comment:
I would put here what is roughly working and especially what's missing in the big picture.